solc/CommandLineInterface.cpp

```diff
@@ -921,6 +930,11 @@ General Information)").c_str(),
 			"Supported Inputs is the output of the --" + g_argStandardJSON + " or the one produced by "
 			"--" + g_argCombinedJson + " " + g_strAst + "," + g_strCompactJSON).c_str()
 		)
+		(
+			"lsp",
+			"Enables Language Server (LSP) mode. "
+			"This won't compile input but serves as language server to to clients."
```

Comment:
It also means that all other options are ignored, is it?

Reply:
For now, yes, it should be mentioned (not ignored but basically disallowed). I am not sure yet if any other flag would make sense, since you usually configure it via the settings object from the client.

p.s.: clangd however allows passing config via CLI flags, but I think in our case, where vscode (hopefully remix?) would be the clients, it's best to stick to the LSP way of passing settings (i.e. via protocol).

Reply:
Please add this piece of information to the description string.